src/main/java/jenkins/plugins/asynchttpclient/AHCUtils.java

```diff
@@ -70,93 +58,4 @@ public static ProxyServer getProxyServer() {
         return proxyServer;
     }
 
-    /**
-     * Return the default {@link HostnameVerifier} to use with {@link AsyncHttpClient}.
-     *
-     * @return the default {@link HostnameVerifier} to use with {@link AsyncHttpClient}.
-     * @since 1.7.24.1
-     */
-    public static HostnameVerifier getHostnameVerifier() {
-        return AHC.acceptAnyCertificate ? new HostnameVerifier() {
-            @Override
-            public boolean verify(String s, SSLSession sslSession) {
-                return true;
-            }
-        } : new DefaultHostnameVerifier();
-    }
-
-    /**
-     * Return the default {@link SSLContext} to use with {@link AsyncHttpClient}.
-     *
-     * @return the default {@link SSLContext} to use with {@link AsyncHttpClient}.
-     * @since 1.7.24.1
-     */
-    public static SSLContext getSSLContext() {
```

Comment:
I have found usages in some proprietary plugins I have access to. I believe this method need a compatible API if possible